azure/components/db/state.ftl

```diff
@@ -4,42 +4,61 @@
 
     [#local core = occurrence.Core]
     [#local solution = occurrence.Configuration.Solution]
+    [#local engine = solution.Engine]
 
-    [#local dbServerId = formatResourceId(AZURE_DB_POSTGRES_SERVER_RESOURCE_TYPE, core.ShortName)]
+    [#switch engine]
+        [#case "postgres"]
+            [#local server_resource_type = AZURE_DB_POSTGRES_SERVER_RESOURCE_TYPE]
+            [#local server_database_resource_type = AZURE_DB_POSTGRES_SERVER_DATABASE_RESOURCE_TYPE]
+            [#local server_vnet_rule_resource_type = AZURE_DB_POSTGRES_SERVER_VNET_RULE_RESOURCE_TYPE]
+            [#local server_config_resource_type = AZURE_DB_POSTGRES_SERVER_CONFIGURATION_RESOURCE_TYPE]
```

Comment:
The convention used throughout Hamlet for variable names is not documented at all, but it'd be good to keep to it for consistency:
* camelCase for local and global vars
* ALL_CAPS_WITH_UNDERSCORES for data types - this is deliberate to stand out easily

There is a lot of boilerplate when using Freemarker so the reduction in line-length through use of camel-case is helpful

Reply:
Cool, thanks